src/test/scala/grading/Lab2Tests.scala

```diff
@@ -53,61 +53,85 @@ class Lab2Grader extends JUnitSuite {
 
         val tests = List[CPUTestCase](CPUTestCase("add1",
                       Map("single-cycle" -> 1),
+                      Map(), 
+                      Map(),
                       Map(5 -> 1234),
                       Map(0 -> 0, 5 -> 1234, 6 -> 1234),
                       Map(), Map()),
           CPUTestCase("addi1",
                       Map("single-cycle" -> 1, "five-cycle" -> 5, "pipelined" -> 5),
+                      Map(), 
```

Comment:
Hmm... I thought five-cycle was removed...

Reply:
These files didn't pop up as conflicts during rebasing. Should I go ahead and remove the five cycle arguments manually? I'm guessing Daniel missed them?